internal/contour/listener_test.go

```diff
@@ -17,14 +17,14 @@ import (
 	"reflect"
 	"testing"
 
-	"github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
you shouldn't need to rename this import

Reply:
I think vscode is trying to be fancy. Let me see if I can turn that off.